portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,36 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Stephs Portfolio (:</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+        <center>
+      <h1>This is Me!</h1>
+      <p>I'm a 21 year old first gen student from LA going to UCSD
+         I'm passionate about outreach and social justice. I'm currently the 
+          outreach co-chair for Women in Computing and want to start a Latinx's in
+          tech mini org. I love computer scinece and problem solving and I also 
+          love/want to learn more about social activism and movements.
+      </p>
+      <p>My plans post graduation include going into industry and <i>ideally</i>
+      working for a company that does community focused work </p>
+        <p> In my free time I like to go down rabbit holes on random subjects,
+            make Mexican food vegan, eating brunch from many of La Jolla's brunch
+            spots, and watching movies and the LA Lakers
+        </p>
+        <p>Feel free to recommend readings on social justice and related topics!</p>
+
+      
+    <img src="images/31.png"/><p>This is a lil me I got at the Google 
+          space at the Grace Hopper Conference 2018!</p>
       <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <button onclick="randomXDQuote()">Random Quote!</button>
```

Comment:
does this log an error when you click the button? I don't see a function named randomXDQuote defined anywhere. Feel free to add an empty function with a comment to implement it in a future PR (pull request) to your script.js file.

Reply:
I realized I didn't add script.js, so it's not updated here :0. I'll add right now!